ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml

```diff
@@ -78,7 +78,13 @@ tests:
       COMPUTE_NODE_TYPE: n2-standard-4
     test:
     - as: win10-pipeline-example-test
-      commands: TARGET=windows10 ./automation/e2e-pipelines/example-pipelines-test.sh
+      commands: |
+        if ! hash kubectl 2>/dev/null; then
+          pushd $(dirname $(which oc))
+          ln -s oc kubectl
+          popd
+        fi
```

Comment:
Can we move handling of this to `./automation/e2e-pipelines/example-pipelines-test.sh` instead? Avoids duplication and keeps the job configuration easier to understand.